PIM/Operation/01_ManageProducts.md

```diff
@@ -0,0 +1,313 @@
+
+  The progress bar in the *Applying Changes* wizard window displays **100%** when the changes has been applied.
+
+  > [Info] As the change job is executed in the background, the wizard window can be closed before the progress bar reaches **100%**.
+
+  ![Applying changes](../../Assets/Screenshots/PIM/Products/List/MassEditingWizard/ApplyingChanges.png "[Applying changes]")
+
+8. Click the [FINALIZE] button in the bottom right corner.   
+  The changes have been saved. The *Applying Changes* wizard window is closed.
+
+
+
+## Copy a product
+
+Use the copy function to duplicate a product and its properties to create a similar product where you only have to adjust individual attributes.
+
+#### Prerequisites
+
+At least one product has been created, see [Create a product](#create-a-product).
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Select the checkbox of the product you want to copy.   
+  The editing toolbar is displayed above the product list.
+
+2. Click the [COPY PRODUCT] button in the toolbar.   
+  The *Edit Product (Copy)* view is displayed. The *Attributes* tab is preselected. All attribute values from the selected product are adopted to the copied product.
+
+  ![Attributes](../../Assets/Screenshots/PIM/Products/List/Attributes/AttributesCopy.png "[Attributes]")
+
+3. Click the *SKU* field and edit the SKU for the new product.
+
+  > [Info] The SKU (Stock Keeping Unit) is an identification number for the product. Therefore, it should be unique within the *PIM* module.
+
+4. Click the  *Product Name* field and edit the product name for the new product.
+
+5. Add or edit the data for the product in the *Attributes* tab. It is recommended to complete at least the *Product Name*, *Price* and *Tax class* fields.   
+
+  > [Info] All fields marked with the note *(required)* are used for the completeness calculation of product attributes. You can enable the *show empty required attributes only* toggle to display only the incomplete required attribute fields for the selected product.
+
+6. Click the [SAVE] button in the upper right corner.   
+  The product has been saved and is added to the product list. The *Edit Product (Copy)* view is closed. The product list is displayed.
+
+
+
+## Create an offer in PIM
+
+Create an offer to a product so you can sell the product in a selected connection. In the *PIM* module you can  create offers to all PIM products. You can either create an offer for a single connection or multiple connections.
+
+### Create a single connection offer in PIM
+
+If you want to offer the product on a single connection only or you have just one connection created, it is recommended to create a single connection offer.
+
+#### Prerequisites
+
+- At least one product has been created, see [Create a product](#create-a-product).    
+- At least one connection has been created.
```

Comment:
Wo? Wie?
In Integration/05_ConfigureChannels.md steht folgende Info, verlinkte Datei existiert aber noch nicht:
[Info] A channel is not equal to a marketplace or web store connection. You can create a channel and use it for another connection. For detailed information about creating a new connection, see [Create a connection](https://github.com/Actindo-AG/Documentation/blob/ACD-315/PIM/Integration/to_be_completed).